doc/Library_Inventory.md

```diff
@@ -0,0 +1,464 @@
+# Azure SDK for .NET Libraries Inventory
+
+## Summary
+
+- Total libraries: 424
+- Management Plane (Swagger): 206
+- Management Plane (TSP): 0
+- Data Plane (Swagger): 151
+- Data Plane (TSP): 0
+- Unknown generator: 67
+
+
+## Data Plane Libraries using TSP
```

Comment:
tsp is detected by the presence of a tsplocation.yaml file.

Reply:
We should split out tsp migrated libraries based on old vs new generator. New generator will have an emitterPackageJsonPath specified in the tspLocation.yaml.

Reply:
the files are tsp-location.yaml, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid.Namespaces/tsp-location.yaml